A sample proposal to help potential speakers (#1150)
writethedocs/www: docs/conf/portland/2020/example-proposal.rst

:template: {{year}}/generic.html

Example talk proposal title
===========================

Substitute a catchy title that promises pratical lessons

Abstract
--------

> This is the content that Write the Docs would post on the conference schedule. The proposal committee uses everything beyond the abstract to evaluate the quality of your proposal.

Reviewer: Suggestion:

> This is the content that Write the Docs would post **to the conference website. In order to evaluate the quality of your proposal, the proposal committee will also take into account all of the info that you provide beyond the abstract.**

> This is the content that Write the Docs would post on the conference schedule. The proposal committee uses everything beyond the abstract to evaluate the quality of your proposal.

Reviewer 2: ```suggestion
This is the content that Write the Docs would post on the conference schedule. To evaluate the quality of your proposal, the proposal committee also takes into account all of the other information that you provide.
```

Include a brief story, typically two to four paragraphs, that describes your personal work experience with the topic. Make sure the abstract appeals to our audience of documentarians. We're suspicious of proposals that look like they belong at other conferences.

Include a list of things that our audience can learn from your talk, such as:

- Lessons that the audience can apply in their own work
- Things that audiences should research further
- Spoilers that provide details about the talk

Help your audience think: "Oh yes, this talk could help me when I do X in my work!"

Avoid walls of text. We recommend that you limit your proposal to a maximum of 300 words. If your proposal is accepted, this is what the audience will see in the program when they preview your talk.

Who and Why
-----------

Use this opportunity to show the selection committee that you empathize with your audience. Help them think: "Oh yes, this talk could help me when I do X in my work!"

> Our audience creates documentation primarily for software. Given the variety of tools used for software documentation, we rarely accept talks that focus on a specific software tool.

Reviewer: Suggestion:

> Our audience creates documentation primarily for software. Given the variety of tools used for software documentation, we are wary about accepting talks that focus on a specific software tool **or set of tools. Talks that discuss tools should also discuss the wider context, applications, and implications of implementing the tools.**

Author: I think this would invite more tooling talks, so I don't think this is wise. However,

@plaindocs If you think a substantial number of tooling talks are inevitable, @eirinikos 's option makes sense.

Reviewer 2: Hmmm, I think a little leeway is a good thing. We could always roll back if it doesn't work out. :-D

> Our audience creates documentation primarily for software. Given the variety of tools used for software documentation, we rarely accept talks that focus on a specific software tool.

Reviewer 2: ```suggestion
Our audience creates documentation primarily for software. Given the variety of tools used for software documentation, we rarely accept talks that focus on a specific software tool or set of tools. Talks that discuss tools should also discuss the wider context, applications, and implications of implementing the tools.
```

Our audence goes beyond the technical writing community. Here's a typical demographic distribution of people who attend our confernces:

* Technical Writers (60%)
* Developers (10%)
* Support Staff (10%)
* Managers (10%)

> * Community Contributors, Enthusiasts & Other Folks (10%)

Reviewer: What do you think of adding an explicit note to the end of this section? e.g.,
`Only the proposal committee will see the info that you include here.`

> 

Reviewer 2: ```suggestion
 Only the proposal committee will see the information that you include here.
 
```

Other Information
-----------------

Based on your background, use this section to describe your qualifications to the selection committee.

We welcome talks from first-time speakers, from industry experts, and from everyone in between. Whatever your background and experience, we prefer hearing about new approaches rather than about tried-and-tested technology. We especially welcome talks from underrepresented groups within the tech community.

We've accepted talks from people who work in a variety of roles, including:

* Writers
* Developers
* Designers
* Testers
* Support
* Developer advocates
* Community managers
* Scientists
* Librarians

> * Teachers

Reviewer: What do you think of adding an explicit note to the end of this section? e.g.,
`Only the proposal committee will see the info that you include here.`

> * Teachers

Reviewer 2: ```suggestion
* Teachers

 Only the proposal committee will see the information that you include here.
```
